src/main/resources/assets/ironchest/lang/fi_FI.lang

```diff
@@ -0,0 +1,177 @@
+
+tile.ironchest.shulker_box.crystal.white.name=Valkoinen Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.orange.name=Oranssi Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.magenta.name=Violetti Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.light_blue.name=Vaaleansininen Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.yellow.name=Keltainen Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.lime.name=Lime Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.pink.name=Pinkki Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.gray.name=Harmaa Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.silver.name=Vaaleanharmaa Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.cyan.name=Syaani Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.purple.name=Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.blue.name=Sininen Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.brown.name=Ruskea Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.green.name=Vihreä Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.red.name=Punainen Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.black.name=Musta Kristallinen Shulker-laatikko
+
+tile.ironchest.shulker_box.obsidian.white.name=Valkoinen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.orange.name=Oranssi Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.magenta.name=Violetti Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.light_blue.name=Vaaleansininen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.yellow.name=Keltainen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.lime.name=Lime Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.pink.name=Pinkki Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.gray.name=Harmaa Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.silver.name=Vaaleanharmaa Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.cyan.name=Syaani Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.purple.name=Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.blue.name=Sininen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.brown.name=Ruskea Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.green.name=Vihreä Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.red.name=Punainen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.black.name=Musta Obsidiaani Shulker-laatikko
+
+item.ironchest.shulker_box.iron_gold.name=Parannus: Rautaisesta Kultaisen Shulker-laatikko
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.gold_diamond.name=Parannus: золотого в алмазный Шалкеровый ящик
```

Comment:
Why russian?

Reply:
I from russian, and me simple translate from russian with check english. I do it at night and didnt see this line

Reply:
7 lines

Reply:
Yesterday I think I translate all. But maybe forget save before close Visual Studio Code